packages/calcite-components/src/components/block/block.scss

```diff
@@ -3,21 +3,42 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
- * @prop --calcite-block-padding: Specifies the padding of the block `default` slot.
+ * @prop --calcite-block-background-color: Specifies the background color of the component.
+ * @prop --calcite-block-border-color: Specifies the border color of the component.
+ * @prop --calcite-block-description-text-color: Specifies the text color of the component's description.
+ * @prop --calcite-block-header-background-color: Specifies the background color of the component's header.
+ * @prop --calcite-block-heading-text-color: Specifies the text color of the component's heading.
+ * @prop --calcite-block-padding: **deprecated** use `--calcite-block-space` instead - Specifies the padding of the component's `default` slot.
+ * @prop --calcite-block-space: Specifies the space of the component's `default` slot.
+ * @prop --calcite-block-toggle-icon-color: Specifies the color of the component's toggle icon.
  */
 
+@import "../../assets/styles/animation";
+@import "../../assets/styles/header";
+
 :host {
-  @extend %component-host;
```

Comment:
You could use the mixin `@mixin base-host()` instead of this if you want to keep the box-sizing.